dart_cid.iml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should this file be renamed to: `dart_multihash.iml` ?

Reply:
Fixed on https://github.com/dwyl/dart_multihash/commit/15bcf43e1667b8c6c79bbae7f545085f229f8af6